packages/i18n/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Add new `addLocaleData` method to merges locale data into the Tannin instance by domain. Note that this function will also merges the domain configuration.
+
```

Comment:
Let's fix the typo in "merges" which should read as "merge". And I would personally remove the "Note" because a changelog entry doesn't need this level of detail about the new method.

The PR will be finally ready to land after this last fixup is done.

Reply:
Oops...I'm sorry for the typo 😅. 

> And I would personally remove the "Note" because a changelog entry doesn't need this level of detail about the new method.

Okay 👍